### DISCUSSION TITLE
How to setup and start ?

### DISCUSSION THREAD

**Discussion by simulatorwinner** [adepierre/Botcraft#15] [Q&A]
I've git cloned, etc, etc. I've setup it completely with cmake, now I want to start the bot, how do I start it, and get it on my world with Lan, so I can watch it and on my other moniter I can see the bots view. How can I do that ?

I ran the cmd "g++ -o main main.cpp" in cmd prompt the main.cpp is from Hello World example, but I got Fatal error:  

main.cpp:4:10: fatal error: botcraft/Game/InterfaceClient.hpp: No such file or directory
 #include "botcraft/Game/InterfaceClient.hpp"
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.

**Comment by adepierre** [comment]
I don't know exactly what you mean by "I've setup completely with cmake", but after the ``make all`` step, you should already have a compiled version of the HelloWorld program that you can launch.

**Reply by simulatorwinner** [reply]
ok
